src/libs/Navigation/AppNavigator/PublicScreens.js

```diff
@@ -29,7 +29,7 @@ export default () => (
         <RootStack.Screen
             name="SetPassword"
             options={defaultScreenOptions}
-            component={SetPasswordPage}
+            component={SetPasswordForm}
```

Comment:
Not sure why we are renaming this? Can we just change it back to `SetPasswordPage` it's a page not a form?